downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1563 @@
+
+```ballerina
+string str = "Ballerina Programming Language";
+boolean contains = str.includes("Language", 10);
+```
+
+###### Introduction of the `sleep` Method
+
+A new `sleep` method, which pauses the execution of the current strand for a specified time in seconds has been added to the `ballerina/lang.runtime` library.
+
+```ballerina
+runtime:sleep(2.0);
+```
+
+###### Introduction of the `getStackTrace` Method
+
+A new `getStackTrace` method has been introduced in the `ballerina/lang.runtime` library to get a stack trace for the current call stack for the specified execution point. It returns an array of stack frames. 
+
+```ballerina
+runtime:StackFrame[] stackFrames = runtime:getStackTrace();
+```
+
+###### Introduction of the `Cloneable` Type
+
+A new `Cloneable` type has been introduced to the `ballerina/lang.value` library. This is a cyclic union type. This type represents the values on which `clone` and `cloneReadOnly` can be applied on.
+
+```ballerina
+public type Cloneable readonly|xml|Cloneable[]|map<Cloneable>|table<map<Cloneable>>;
+```
+
+#### Rename of the `ballerina/java` Module
+
+The `ballerina/java` module is renamed to `ballerina/jballerina.java`.
+
+#### Runtime
+
+New APIs are added for creating immutable Ballerina records and arrays from Java. 
+
+The new `ValueCreator.createReadonlyArrayValue` API creates a new `readonly` Ballerina array from the given Java array and the `ValueCreator.createReadonlyRecordValue` creates a new `readonly` Ballerina record.
+
+##### Configurable
+
+The `configurable` feature is improved to support variables with decimal and arrays of int, float, string, boolean and decimal types.
+
+Configurations can be provided at runtime using the `Config.toml` file in the current working directory or by exporting the file path using the `BALCONFIGFILE` environment variable. For the tests, configurations can be overridden by having the `Config.toml` file inside the tests directory.
+
+#### Developer Tools
+
+##### Language Server
+
+- `Find all Reference` Support
+- CodeLens support for `DocumentThis` is available to auto-generate documentation for public documentable constructs
+- CodeAction for `ChangeVariableType` is available to solve incompatible variable assignments
+
+##### Project API
+
+TOML and MD files are made a part of the Project API.
+
+##### Debugger
+
+- Enhanced table variable presentation with the support to view child entries
+- Introduced type test expression evaluation support
+
+##### Test Framework
+
+Support for function pointers in the `@test:Config {}` annotation. This applies to the `before`, `after`, `dependsOn`, and `dataProvider` fields of the annotation. 
```

Comment:
Should we mention that this previously expected the name of the function (a string)? @Dilhasha @Ibaqu

Reply:
Will the following suggestion be enough?

```suggestion
Support for function pointers in the `@test:Config {}` annotation. The fields `before`, `after`, `dependsOn` and `dataProvider`, which previously expected the name of the function as a `string`, now accept function pointers instead.
```